websites/D/Diziroll/presence.ts

```diff
@@ -0,0 +1,54 @@
+const presence = new Presence({ clientId: "818550584994168934" });
+
+let stream: { duration: number; currentTime: number; paused: boolean };
+presence.on("iFrameData", (data: { duration: number; currentTime: number; paused: boolean }) => (stream = data));
+
+presence.on("UpdateData", async () => {
+  const path: string = document.location.pathname,
+  presenceData: PresenceData = {largeImageKey: "diziroll"};
+  presenceData.startTimestamp = Date.now();
+  if(path.startsWith("/arsiv")){
+  presenceData.details = "Bir sayfaya bakıyor:";
+  presenceData.state = "Arşiv";
+
+  }else if(path.startsWith("/listeler")){
+  presenceData.details = "Bir sayfaya bakıyor:";
+  presenceData.state = "Listeler";
+  
+  }else if(path.startsWith("/hesabim")){
+      presenceData.details = "Bir sayfaya bakıyor:";
+      presenceData.state = "Hesabım";
+
+  }else if(path == "/"){
+      presenceData.details = "Bir sayfaya bakıyor:";
+      presenceData.state = "Ana Sayfa";
+
+  }else if(document.getElementById("archive-page")){
+      presenceData.details = "Bir dizi türünü inceliyor: ";
+      presenceData.state = document.querySelector("div.title").innerText;
```

Comment:
```suggestion
      presenceData.state = document.querySelector("div.title").textContent;
```

Preferred to use textContent, most presences do this :)